### DISCUSSION TITLE
Change Login or Update Login

### DISCUSSION THREAD

**Discussion by TealCanadyRD** [community/community#58286] [Codespaces] [answered]
### Select Topic Area

Bug

### Body

Hey,
I'm running Debian Linux and experienced a crash on VScode. Since that time I attempted to go back to the code space but none appear. I've recreated the codespace using the same account; still nothing appears. I've uninstalled the extension and reinstalled. I've restarted and attempted to CTRL+SHIFT+P -> Codespace: Sign In and that does nothing it would seem. I've wiped the cache for VSCode and had no luck. 

I'm running 1.79.2 of VSCode so it could be an issue with that version but I have no way to simply logout of Codespace in VSCode. I've even tried to update my password in the hopes it would prompt me to reset again with no luck.  I can get to the code online using the same login and attempted to launch from the Codespace site to VScode produces this error: 

Connect to Codespace: Get codespace request failed with status 401. Correlation ID: 9018:4DC4:F01F3F:F8D6E7:648D0023

Just really frustrating; I guess my last hope is to reinstall VSCode and hope it let's me renew my credentials.  Any help would be great but a feature would be great as well if I wanted to logout of out Codespace for one client to log into a different client. 

Cheers,

Teal

**Comment by the-officialjosh** [comment, marked as answer]
Here are a few steps you can try to resolve the issue:

1. **Clear VSCode cache**: You mentioned wiping the cache for VSCode, which is a good first step. Make sure to completely remove any remnants of the previous installation before reinstalling.

2. **Check your network connection**: Verify that you have a stable internet connection and that there are no network issues affecting your access to Codespaces. Sometimes network disruptions can cause connection problems.

3. **Update VSCode**: If you suspect that the issue may be related to the version of the VSCode you're using (1.79.2), consider updating to the latest version. Newer releases often include bug fixes and improvements.

4. **Consider trying an older version of VSCode**: If updating doesn't resolve the problem, you can also try installing an older version of VSCode to see if it works with Codespaces. You can find older versions on the VSCode release page.

5. **Contact Microsoft support**: If none of the above steps solves the issue, it would be best to reach out to Microsoft support for further assistance. They can provide more specific guidance and help troubleshoot the problem with Codespaces.

Regarding logging out of Codespaces, currently, there isn't a built-in feature in VSCode to log out of Codespaces separately. It's recommended to sign out of your GitHub account in VSCode or use a different GitHub account to access Codespaces.

I hope these suggestions help you resolve the issue you're facing with Codespaces in VSCode.

**Reply by TealCanadyRD** [reply]
Thank you i will try these suggestions and let you know

**Reply by TealCanadyRD** [reply]
Killing the entire ~/.config/Code folder after uninstalling vscode; then reinstalling vscode did the trick. Thanks for the advice; I would like to add in that hopefully a feature can be put in to log out of codespaces in vscode. I can see contractors being forced to use a specific email / domain in certain situations where this could be problematic. Thanks again @the-officialjosh !

**Reply by the-officialjosh** [reply]
You're welcome glad I could help

**Reply by tytyieq** [reply]
%20or%201=1#

**Reply by tytyieq** [reply]
'or 1=1--

**Reply by tytyieq** [reply]
<SVg/ONloAd = alert(1);>

**Reply by tytyieq** [reply]
<DIV STYLE="background-image: url(&#1;jaVascRipt:alert('XSS'))">

**Reply by tytyieq** [reply]
<IMG SRC=&#106;&#97;&#118;&#97;&#115;&#99;&#114;&#105;&#112;&#116;&#58;&#97;&#108;&#101;&#114;&#116;&#40;&#39;&#88;&#83;&#83;&#39;&#41;>

**Reply by tytyieq** [reply]
<IMG SRC=&#106;&#97;&#118;&#97;&#115;&#99;&#114;&#105;&#112;&#116;&#58;&#97;&#108;&#101;&#114;&#116;&#40;&#39;&#88;&#83;&#83;&#39;&#41;>

**Reply by tytyieq** [reply]
%E5%E98%8A

**Reply by tytyieq** [reply]
%BCscript%BEalert(%A2XSS%A2)%BC/script%BE
